cmd/verify.rb

```diff
@@ -68,46 +67,8 @@ def run
                 next
               end
               bottle.fetch
-              result = Homebrew::Attestation.check_attestation(bottle, "Homebrew/homebrew-core")
-              if result[:verified]
-                if args.json?
-                  json_results.push(result[:data])
-                else
-                  ohai "Verified #{bottle.name} with tag #{bottle_tag}."
-                end
-              else
-                case result[:error]
-                when JSON::ParserError
-                  opoo "#{bottle.name} with tag #{bottle_tag} returned invalid json: #{result[:message]}"
-                  next
-                when Homebrew::Attestation::COMMAND_ERROR
-                  opoo "#{bottle.name} with tag #{bottle_tag} unverified, checking backfill signature."
-                  backup_result = Homebrew::Attestation.check_attestation(bottle, "trailofbits/homebrew-brew-verify")
-                  if backup_result[:verified]
-                    timestamp = backup_result.dig(:data, 0, "verificationResult", "verifiedTimestamps", 0,
-                                                  "timestamp")
-                    if timestamp.nil?
-                      opoo "Unable to verify #{bottle.name} with tag #{bottle_tag} " \
-                           "because json does not have signature timestamp in expected location."
-                      next
-                    end
-                    parsed_timestamp = DateTime.parse(timestamp)
-                    # Backfilled signatures all occured before this date so if we find
-                    # a backfilled signature after this date, we should disregard it.
-                    last_signature_date = DateTime.new(2024, 3, 14)
-                    if parsed_timestamp < last_signature_date
-                      if args.json?
-                        json_results.push(backup_result[:data])
-                      else
-                        ohai "Verified #{bottle.name} with tag #{bottle_tag} using backfill signature."
-                      end
-                    else
-                      opoo "Unable to verify #{bottle.name} with tag #{bottle_tag}. " \
-                           "Backfilled signature dated after last backfill date."
-                    end
-                  end
-                end
-              end
+              attestation = Homebrew::Attestation.check_core_attestation bottle
```

Comment:
Nitpick: maybe catch the `AttestationError` that happens here and convert it into something a little more human-friendly?

Reply:
Done, I think `odie` is the correct option here?

Reply:
Actually went back to `opoo` because we don't actually die if there was an error.

Reply:
Hmm, I think we want to change that behavior and actually die if there's an error (that way people can script with this subcommand). But we can iterate on that.